src/AI/agents/benchmarks/bootstrap_api_key.py

```
@@ -0,0 +1,104 @@
+    print(f"Wrote AGENT_DESIGNER_API_KEY to {env_path}")
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("--username", default="localgiteaadmin")
+    parser.add_argument("--name", default="benchmark-runner")
+    parser.add_argument("--write-env", action="store_true")
+    args = parser.parse_args()
+
+    account_id = _sql(
+        f"SELECT id FROM designer.user_accounts WHERE username = '{args.username}';"
```

Ruff (0.16.0)

[error] Possible SQL injection vector through string-based query construction

(S608)